src/wovnio/wovnphp/Store.php

```diff
@@ -204,9 +220,13 @@ public function convertToCustomLangCode($lang_code)
         return $lang_code;
     }
 
-    public function convertToOriginalCode($lang_code)
+    public function convertToOriginalCode($lang_code, $customLangAliases = null)
```

Comment:
Don't like this null default parameter, but I think the real problem is `updateSettings` taking in this $vals parameter. From what I can tell, $vals is expected to be the same object as this->settings. I would much rather store using this->settings always, if it is possible.

Reply:
Maybe it could be done by doing `$this->settings = array_merge($defaultSettings, $userSettings);` first in the constructor, followed by `updateSettings`, then you wouldn't need $vals.

Reply:
I agree, I'll see what can be done.

Reply:
I fixed the initialization of Store so that my change can expect settings to be usable instead of adding to method prototype.